### PR TITLE
copied recipe finder to myrecipe page

### PR DESCRIPTION
**Pull request by cchchoi1986**

### CODE REVIEW

**Comment by cchchoi1986** [comment]
I just copied the recipe finder page to my recipe page.... changed the title and hid the filtering engine on load versus the finder page.

did little css on the pagination to fit into our design

**Comment by cchchoi1986** [comment]
@harrychenca @paulld

File: app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb [hunk 1]
```diff
@@ -127,19 +127,19 @@ app.controller('RecipeFinderCtrl', ['$scope', '$http', '$routeParams', function(
...
       $(".hide-finder-button").attr('data-status',"closed");
     } else {
       $("div.filter").show();
       $(".hide-finder-button").attr('data-status',"open");
       $(".filter-results").addClass("col-sm-6");
       $(".reset-button").show();
-      $(".submit-button").show();
-      $(".hide-finder-button").removeClass("col-xs-6");
-      $(".create-recipe-button").removeClass("col-xs-6");
+      // $(".submit-button").show();
```
Comment (paulld) [inline app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb:140]: you can remove all that if they're not necessary anymore

File: app/assets/templates/pages/myRecipes.html [hunk 2]
```diff
@@ -1,51 +1,67 @@
...
+            <option>Ingredient</option>
+            <option>Keyword</option>
+          </select>
+          <button type="submit" class="add-keyword-button btn btn-default col-xs-2">Add</button>
+        </form>
 
         <!-- filter by meals -->
         <div class="filter-meals-wrapper row">
-          <div class="filter-meals-heading col-xs-12 col-md-3">Meal:
+          <div class="filter-meals-heading col-xs-12"> <i class="fa fa-cutlery"></i> Meals
```
Comment (paulld) [inline app/assets/templates/pages/myRecipes.html:58]: it would be cleaner to put `<i class="fa fa-cutlery"></i> Meals` in a separate line

**Pull request merged**